packages/design-system/src/interfaces/badge.d.ts

```diff
@@ -0,0 +1,5 @@
+interface IBadgeProps extends React.HTMLAttributes<HTMLSpanElement> {
+  secondary?: boolean;
+}
+
+export const Badge: React.FC<IBadgeProps>;
```

Comment:
did you check is the interface works correctly?

Reply:
Yes, it does.